config.py

```diff
@@ -13,13 +13,13 @@ def set_region(region):
     REGION = region
 
 
-def load(env=None):
+def load():
     """
-    Load settings
+    Loads the correct settings file based on FLASK_ENV.
     """
-    env = env or os.environ.get("FLASK_ENV", "development")
+    flask_env = os.environ["FLASK_ENV"]
```

Comment:
What happens if `FLASK_ENV` isn't set?

Reply:
A very readable exception happens, as it should.